Deliverables/R1 - April 16.md

```diff
@@ -0,0 +1,15 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+
+This week I was able to add a Swanky new animation to know breaking process thanks to Conair reminding me that linear interpolation is a thing. I was also able to create backstage menus for my inventory screen, my overview screen, and my structures screen as outlined in my original GitHub issue. The inventory screen is now a much cleaner and scalable solution to handling inventory in the tracking of materials than what I was previously using as it contains all that information within a compound data structure as opposed to a collection of global variables. Additionally, while I don't have a measurable goal to display this, I also learned how GML handles data structures, variable scoping, enumeration, and partial Sprites which greatly streamlines and simplifies my workflow moving forward.
+
+## 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+
+That leads into the main challenges and difficulties that I faced this week of not initially knowing how GML handled complex data structures, I did eventually get through it thanks to a lot of research, yo-yo's documentation, and several helpful YouTube videos. Though I am still a little bit confused on when I need to be using instance vs local variables, but I imagine that's the kind of thing I'll figure out with more experience.
+
+## 3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+
+Over the next week, I intend to implement my structures backstage menu as well as gain that functionality for it. What that means is I need to populate the backstage menu with structure objects, these objects need to be clickable and that should allow me to create them on the game map, and the game map creation process should allow them to snap to the pre-existing hexagonal grid. I expect each of those individual tasks will take a full workday to complete.
```

Comment:
Great goals. Writing UI code is time-consuming for sure, and I imagine the GMS tools you have been using don't make it particularly easy, especially after reviewing your code. Keep chippin' away at this, you've got this down.